### DISCUSSION TITLE
Cant run compiled demo file (default server example): "run again with --allow-net", but I'm actually using the flag.

### DISCUSSION THREAD

**Discussion by ibnz36** [denoland/deno#9973] [Q&A] [answered]
Hi. I was trying demo by the first time, and I already have some issues.
I really like the fact that demo can be compiled, so, I tried compiling the default server example.

Here's my code, the command and the error:

`demo-test.ts`
```ts
import { serve } from "https://deno.land/std@0.92.0/http/server.ts";
const server = serve({ port: 8000 });
console.log("http://localhost:8000/");
for await (const req of server) {
  req.respond({ body: "Hello World\n" });
}
```

I compiled using `demo compile --unstable demo-test.ts --allow-net`,
And runned the executable using `./deno.test --allow-net`

It gives me this error:

```log
error: PermissionDenied: Requires net access to "0.0.0.0:8000", run again with the --allow-net flag
```

**Comment by Soremwar** [comment, marked as answer]
Hi there @Brian3647 

Just like when using the `deno run` subcommand, you need to pass all flags before the script name. `deno compile --unstable --allow-net demo-test.ts` will work
